### DISCUSSION TITLE
Conversion of Z values

### DISCUSSION THREAD

**Discussion by MohitDahliya** [ARM-DOE/pyart#1281] [Q&A] [answered]
How to convert Z in dBZ to mm6m-3  ?

**Comment by zssherman** [comment, marked as answer]
Hi @MohitDahliya,

To go to linear Z use:
 z_lin = 10.0**(radar.fields['reflectivity']['data']/10.0)

Cheers

**Comment by MohitDahliya** [comment]
Yes but if you take like this, that means you are taking Zo = 1. So is that
correct..?

**Reply by rcjackson** [reply]
Yes, this conversion assumes Zo = 1 mm^6 m^-3.

**Reply by rcjackson** [reply]
![image](https://user-images.githubusercontent.com/22085748/193601734-85431f76-0c33-4e41-b58d-45f7059e8d92.png)

**Comment by MohitDahliya** [comment]
But Is Z really around 1 mm^6mm^-3 ? Because I am getting some under values
using this formula as it is...

**Reply by mgrover1** [reply]
> But Is Z really around 1 mm^6mm^-3 ? Because I am getting some under values using this formula as it is...
> […](#)
> On Mon, 3 Oct 2022, 19:54 Bobby Jackson, ***@***.***> wrote: [image: image] <https://user-images.githubusercontent.com/22085748/193601734-85431f76-0c33-4e41-b58d-45f7059e8d92.png> — Reply to this email directly, view it on GitHub <[#1281 (reply in thread)](https://github.com/ARM-DOE/pyart/discussions/1281#discussioncomment-3787979)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AQACDYUN5EQFYUCNYOUXXELWBLUC3ANCNFSM6AAAAAAQ25YWS4> . You are receiving this because you were mentioned.Message ID: ***@***.***>

This is the official American Meteorological Society (AMS) glossary definition and method as well. We typically use the community standards/peer reviewed standards, which in this case, is the AMS.
- [AMS Reference](https://glossary.ametsoc.org/wiki/Dbz)

**Comment by MohitDahliya** [comment]
Okay. Thanks a lot !
